spectral_cube/spectral_cube.py

```diff
@@ -2442,6 +2451,8 @@ def _gsmooth_image(args):
                                                            )
                                        )
                                   ])
+        if self.abort:
+            return
```

Comment:
Could we instead return a partial result here?

Reply:
Yes we can, where can. Which variable would be appropriate for this?